playbooks/byo/config.yml

```diff
@@ -1,2 +1,14 @@
 ---
+- name: Verify Requirements
```

Comment:
IIRC, we use `playbooks/byo` only as entry points. The verify code should go under `playbooks/common`. @mtnbikenc ?